### PR TITLE
Track locations via card list

### PR DESCRIPTION
**Pull request by ofekzicher**

### CODE REVIEW

**Comment by eilatc** [comment]
Hi Ofek,
Thanks for contributing!
Can you please elaborate more on this feature?

**Comment by ofekzicher** [comment]
> Hi Ofek,
> Thanks for contributing!
> Can you please elaborate more on this feature?

Sure! I talked with Ofer about this feature - it contains a new page that lets you to look up and filter reports - just like you do with flights. You can filter by the name of the report, and also look for reports in specific time ranges (i.e all the reports in Haifa from 01/03/2020 to 10/03/2020).

File: public/js/app.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /* eslint-disable no-unused-vars, no-undef */
-let map, infoWindow, govData, data, threeDaysButton, allDaysButton, oneWeekButton, twoWeekButton, intervalId;
+let map, infoWindow, data, threeDaysButton, allDaysButton, oneWeekButton, twoWeekButton, intervalId;
 let currentPositionMarker = null;
+var govData;
```
Comment (oferb) [inline public/js/app.js:4]: Revert

File: public/languages/en.js [hunk 2]
```diff
@@ -1,36 +0,0 @@
-const en = {
```
Comment (oferb) [inline public/languages/en.js]: Please check you didn't delete the file

**Review by oferb — APPROVED** [review]

**Comment by oferb** [comment]
What's the status on this?

**Comment by ofekzicher** [comment]
> What's the status on this?

Sorry for all the delay, made a few fixes and it is now merged

**Pull request merged**